bigchaindb/web/views/base.py

```diff
@@ -1,18 +1,19 @@
 """
 Common classes and methods for API handlers
 """
+import logging
 
 from flask import jsonify, request
 
+logger = logging.getLogger(__name__)
+
 
 def make_error(status_code, message=None):
     if status_code == 404 and message is None:
         message = 'Not found'
-
-    response = jsonify({
-        'status': status_code,
-        'message': message
-    })
+    response_content = {'status': status_code, 'message': message}
+    logger.error('HTTP API error: %(status)s - %(message)s', response_content)
```

Comment:
Yea, I'm not sure it's a good idea though, why not just add configuration in `bigchaindb/__init__.py` for Gunicorn's own logging? http://docs.gunicorn.org/en/stable/settings.html#logging